### PR TITLE
first algorithm

### LINKED ISSUE #6: first algorithm
We need an algorithm to vote for one person in one roll (simple, single-transferrable-vote)

### PR DESCRIPTION
**Pull request by simonleeADL**
Closes #6

### CODE REVIEW

**Review by vmharvey — APPROVED** [review]
very clever! happy to see this merged once #13 is merged.

**Review by ibarnsley — APPROVED** [review]
Looks good to me! Happy to merge this once you have added your new updates

**Pull request merged**
